tests/rest/client/test_auth.py

```diff
@@ -465,9 +465,11 @@ def test_ui_auth_via_sso(self) -> None:
           * checking that the original operation succeeds
         """
 
+        fake_oidc_provider = self.helper.fake_oidc_server()
```

Comment:
```suggestion
        fake_oidc_server = self.helper.fake_oidc_server()
```

(and all references -- there's also other instances in this file)